lightly/cli/config/get_config.py

```diff
@@ -0,0 +1,9 @@
+from pathlib import Path
```

Comment:
I'm not very familiar with `lightly` package layout, it looks a bit surprising that this is in the `cli` submodule. Briefly looking at the folder structure `utils` could be suitable. But I believe you know it better so I'll trust your judgement 🙂

Reply:
The `config.yaml` is in the config directory, thus I also put the function here.